boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp_iron.dts

```diff
@@ -22,15 +31,40 @@
 		#size-cells = <1>;
 
 		cpuapp_slot0_partition: partition@2c000 {
-			reg = <0x2c000 DT_SIZE_K(480)>;
+			reg = <0x2c000 DT_SIZE_K(736)>;
+		};
+
+		cpuppr_code_partition: partition@e4000 {
+			reg = <0xe4000 DT_SIZE_K(64)>;
+		};
+
+		cpuflpr_code_partition: partition@f4000 {
+			reg = <0xf4000 DT_SIZE_K(48)>;
+		};
```

Comment:
FYI, I intend to introduce a "memory map" DTS file that both APP and RAD use soon.

This is so that the app can send the correct vector table address to IRONside SE.

We need to make sure we don't conflict with each other more than necessary somehow.

Reply:
I suggest your change goes in first, then I adapt to that. I need this for the random numbers anyway.